newscript.go

```diff
@@ -134,141 +162,83 @@ func goCompatibility() error {
 
 func goGet() error {
 	fmt.Printf("--------Getting u-root \n")
-	cmd1 := exec.Command("go", "get", "github.com/u-root/u-root")
-	err := cmd1.Run()
-	//TODO: Figure why this ^ isn't working
+	cmd := exec.Command("sh", "-c", "export", "GOPATH=\"$HOME/go\"")
```

Comment:
this is totally not how you do it. All you're doing here is running a shell that set its own environment. As soon as that shell exits, that change is done. Just remove lines 165 to 170. They're not needed.